language/Spanish/strings.po

```diff
@@ -246,11 +246,11 @@ msgstr "Red global\tImperfecta\n"
 
 #: src/frontend/mame/ui/selgame.cpp:1110
 msgid "Timing\tUnimplemented\n"
-msgstr ""
+msgstr "Cronometraje\tNo implementado"
 
 #: src/frontend/mame/ui/selgame.cpp:1112
 msgid "Timing\tImperfect\n"
-msgstr ""
+msgstr "Cronometraje\tImperfecto"
```

Comment:
Same here, it will screw up the general info box if the final newline isn't present.